src/main/java/net/minecraftforge/client/model/animation/Parameters.java

```diff
@@ -0,0 +1,90 @@
+package net.minecraftforge.client.model.animation;
+
+import com.google.common.base.Objects;
+
+/**
+ * Various implementations of IParameter.
+ */
+public class Parameters
+{
+    public static enum NoopParameter implements IParameter
```

Comment:
This confused me when I saw it in context. I honestly thought it was a Null Parameter from it's name.

Can I suggest this be PassthroughParameter or something to that effect.